### ISSUE TITLE
Matplotlib dependency in requirements?

### ISSUE THREAD

**Issue by pnkraemer** [unifyai/ivy#680]
Hi! 

I was trying to play around with `ivy` a little bit, and I noticed that the installation experience could be improved by maintaining a more complete list of dependencies. When I `pip install ivy-core` into an empty environment, and I try to `import ivy`, I get `ModuleNotFoundError: No module named 'matplotlib'`. 

It is not that I am not able to resolve this error (well, `pip install matplotlib` does the trick), but I think it would be great if the quickstart would be self-contained. :) 

It seems that an easy fix would be to list matplotlib in the ![requirements file](https://github.com/unifyai/ivy/blob/master/requirements.txt). What do you think? I can open a PR if that helps.

**Comment by jkeane508** [comment]
Hi!

I believe this is no longer an issue with the current version of Ivy on the repo🚀

Thanks 😀

**Comment by vamsivallepu** [comment]
No. The issue still exist. When I installed ivy (the latest version) today, the error is still raising until we install matplotlib manually. Listing matplotlib in the [requirements file](https://github.com/unifyai/ivy/blob/master/requirements.txt) is the only fix I think.

**Comment by jkeane508** [comment]
> No. The issue still exist. When I installed ivy (the latest version) today, the error is still raising until we install matplotlib manually. Listing matplotlib in the [requirements file](https://github.com/unifyai/ivy/blob/master/requirements.txt) is the only fix I think.

Hi 

@vamsivallepu can I ask are you using the version of Ivy on PyPi or the version of Ivy from the GitHub Repo? 😀

Thanks!

**Comment by vamsivallepu** [comment]
The one on the PyPi.

**Comment by jkeane508** [comment]
> The one on the PyPi.

Ok yes we are aware that this is an issue on the current PyPi version of Ivy. We are working super hard on our next release and I can confirm that this has been resolved. 💪

In the meantime, if you like you can use the version of Ivy on the GitHub Repo however this is in rapid development and is changing all the time 🚀

Thanks for Flagging!

**Comment by vamsivallepu** [comment]
Oh ! Thanks for the explanation.
